explorer/src/lib/components/navbar/styles.css

```diff
@@ -0,0 +1,133 @@
+.dusk-navbar {
+  overscroll-behavior: contain;
+  height: 100%;
+  z-index: 999;
+  margin: 0 auto;
+  padding: 25px 0;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.dusk-navbar--menu-hidden {
+  background-color: var(--light-grey);
+  position: sticky;
+  height: auto;
+  min-height: 0;
+}
+
+.dusk-navbar__logo {
+  display: flex;
+  align-items: center;
+}
+
+.dusk-navbar__logo img {
+  width: 86px;
+  height: 20px;
+}
+
+.dusk-navbar__toggle {
+  background-color: transparent;
+  display: inline-flex;
+  align-items: center;
+  font-size: 1rem;
+  border: none;
+}
+
+.dusk-navbar__toggle:focus {
+  outline: none;
+}
+
+.dusk-navbar__toggle-svg {
+  width: 1.875rem;
+  height: 1.875rem;
+  stroke: var(--smokey-black);
+}
+
+.dusk-navbar__menu {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 100%;
+  height: calc(100vh - 80px);
```

Comment:
these `80px` seems a magic number here: is there some relation to another measure? is it better expressed in `rem`?
I would add a comment if there's no explicit relation, to explain why it's used.

Also are you sure of using `vh` and not `lvh` / `svh`?

Reply:
Left a comment explaining the magic number. I do not see a reason not to use `vh`.